.test-infra/jenkins/job_PreCommit_Python_ValidatesRunner_Flink.groovy

```diff
@@ -21,16 +20,19 @@ import PrecommitJobBuilder
 
 // This job runs the suite of ValidatesRunner tests against the Flink runner.
 PrecommitJobBuilder builder = new PrecommitJobBuilder(
-        scope: this,
-        nameBase: 'Python_PVR_Flink',
-        gradleTask: ':beam-sdks-python:flinkValidatesRunner',
-        triggerPathPatterns: [
-                '^model/.*$',
-                '^runners/.*$',
-                '^sdks/python/.*$',
-                '^release/.*$',
-        ]
+    scope: this,
+    nameBase: 'Python_PVR_Flink',
+    gradleTask: ':beam-sdks-python:flinkValidatesRunner',
+    triggerPathPatterns: [
+      '^model/.*$',
+      '^runners/core-construction-java/.*$',
+      '^runners/core-java/.*$',
```

Comment:
We should also add 
`'^runners/extensions-java/.*$'`
`'^runners/reference/.*$'`

Reply:
Why the reference runner should trigger a Flink build I don't see any relation between it and the Flink runner?

Reply:
Oh I see now, it is used for Flink's Job server, shouldn't we get this code out of Reference now that it is getting deprecated?

Reply:
done